marimbabot_simulation/README.md

```diff
@@ -0,0 +1,32 @@
+# Simulation package
+
+## Launch files
+
+There are the following launch files:
+
+* `marimbabot_gazebo.launch`: Launches the simulation in Gazebo.
+* `marimbabot_full_sim.launch`: Launches the simulation in Gazebo, as well as nodes to detect contacts and publish the contact points, and generate MIDI.
+
+
+## How to run
+
+Install dependencies (if not already installed):
+
+```bash
+sudo apt-get install libportaudio2
+sudo apt-get install fluidsynth
```

Comment:
you can remove fluidsynth, it is already installed here https://github.com/UHHRobotics22-23/marimbabot/blob/6341875dd5ed0e60db247a12c4871d0d73637701/marimbabot_audio/package.xml#L19